guides/v2.1/cloud/onboarding/onboarding-tasks.md

```diff
@@ -70,6 +73,11 @@ When you initially create your project, you can access the Magento Admin panel t
     -   Or you can access a store URL available through the [Project Web Interface]({{ page.baseurl }}/cloud/project/projects.html). For the store URL, add /admin at the end. A login panel displays.
 
 1.  Click the forgot Password link and complete the process to change the password.
-1.  Login to the Admin panel using the username Admin and newly changed password.
-
-If you receive an error, you may need to add admin variables for `ADMIN_EMAIL`, `ADMIN_USERNAME`, and `ADMIN_PASSWORD`. Trial accounts should already have the `ADMIN_EMAIL` variable configured for them.
\ No newline at end of file
+1.  Login to the Admin panel using the default username (Project Owner email address)
+and the newly changed password.
+
+If you receive an error, you might need to add admin variables for
+`ADMIN_EMAIL`, `ADMIN_USERNAME`, and `ADMIN_PASSWORD`. Trial accounts should
```

Comment:
This paragraph has some tense conflicts and a wishy washy feel to it, like we're not sure what they will see. The phrase "should already have" is also hard to think about, especially if you're translating it. And there should be a comma before using "which".